util/chunk/row_container.go

```diff
@@ -428,6 +451,11 @@ type SortedRowContainer struct {
 		// It will get an ErrCannotAddBecauseSorted when trying to insert data if rowPtrs != nil.
 		rowPtrs []RowPtr
 	}
+	errInSortingM struct {
```

Comment:
I think reuse ptrM mutex is ok.. This lock is used to protect the synchronization of the sorting processing. 
So we also don't need the function getErrInSorting(), because all the caller need acquire ptrM.

Reply:
done